database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -14,19 +14,15 @@ class CreateUsersTable extends Migration
     public function up()
     {
         Schema::create('users', function (Blueprint $table) {
-
             $table->id();
             $table->string('name');
             $table->string('email')->unique();
-            $table->string('username')->nullable( );
-            $table->string('location')->nullable( );
```

Comment:
do you intend to remove the username, location, avatar and introduction? Is this not being used by other team member?